### ISSUE TITLE
Blue Screen Of Death On Asus ROG Ally

### ISSUE THREAD

**Issue by cmr333** [Valkirie/HandheldCompanion#730]
Hi, I've been using Handheld Companion for ~4 days now

On the 2nd day of using handheld companion, I left my ROG Ally to download rest of my games then I come back few hours later to check on the progress and I'm faced with BSOD that I couldn't fix including booting safe mode or using system restore

After the BSOD issue, I reset Windows 11 with fresh installation using Rufus. Windows update was updating in the background (There were no Asus related drivers or apps installed by then). While Windows was updating, I was in the process of installing Handheld Companion, once the app was installed and launched. I received another BSOD few seconds after hitting "Start service" within the app before I could even change any settings within the app or install any additional apps on Windows

I ended up doing another Windows reset, this time I went a whole day without Handheld Companion, updated my drivers to latest including Asus apps etc. I really wanted Handheld Companion back so on the 4th day I installed it again. This was yesterday and so far I don't have BSOD

Today I thought I would submit an issue as I couldn't find any open issues with people receiving BSOD

**Comment by CasperH2O** [comment]
@cmr333 sorry to hear about your troubles. We had some BSOD issues a few months back when we released the first version of AutoTDP, but that has been resolved.

Can you please provide the HC version you are using, logfiles from /programfiles/handheldcompanion/logs and if present any .dmp files from HC.

What was the error code on the BSOD?

**Comment by cmr333** [comment]
I was using the latest version (0.16.2.5)

I'm on my 3rd attempt of using Handheld Companion at the moment and haven't had BSOD for ~3 days thus far

Do you still want the log and .dmp files from my current version of Handheld Companion that hasn't got the BSOD yet?

I don't recall if there was a proper error code, in hindsight I should've taken a photo but I do recall it mentioning a file path of "Windows\System32\Logfiles\Srt\SrtTrail.txt" if that helps in any way

------------

Also, I should mention that the only hardware I've changed from stock ROG Ally is that I've upgraded my SSD to "2TB WD SN740"

I don't use any SD cards either, also both times I've had BSOD I had Armoury Crate uninstalled

The first BSOD, I had Armoury Crate uninstalled
The second BSOD, I never installed Armoury Crate
3rd attempt of Handheld Companion, I uninstalled Armoury Crate

------------

When I had my first BSOD, I suspected it could be Handheld Companion that was the cause but at the back of my mind I also thought maybe it's the SSD

However when the second BSOD happened seconds after hitting "Start Service" on Handheld Companion, my suspicion for Handheld Companion being the cause of the issue increased significantly

**Comment by CasperH2O** [comment]
@cmr333 only a log of when the BSOD occurred would be helpful really. 

I'll keep this open another week or so, if you have no further issues we can close it.

**Comment by cmr333** [comment]
Update - It's been a while, I haven't had a 3rd BSOD since I've made the ticket. I think it might be long enough to close this ticket

I ended up having a new theory that I think on my 1st and 2nd BSOD, my VRAM was set to auto. On my 3rd attempt, it was set to default 4GB on the VRAM. I think there could be a possible issue with Auto VRAM and Handheld Companion but I didn't want to test my theory out since it's working fine at the moment
